README.md

```diff
@@ -53,3 +53,25 @@ return [
 
 You will see a debugger toolbar showing at the bottom of every page of your application.
 You can click on the toolbar to see more detailed debug information.
+
+
+Open Files in IDE
+-----
+
+You can create a link to open files in your favorite IDE with this configuration:
+
+```php
+return [
+    'bootstrap' => ['debug'],
+    'modules' => [
+        'debug' => [
+            'class' => 'yii\debug\Module',
+            'traceLink' => '<a href="phpstorm://open?url={file}&line={line}">{file}:{line}</a>',
+        ],
+        // ...
+    ],
+    ...
+];
+```
+
```

Comment:
@samdark In fact, there is no a list of possible settings per IDE.

The user is who sets the string when it does the setup in the operating system. There is a link that talks about this in the example I quoted: https://pla.nette.org/en/how-open-files-in-ide-from-debugger